mti_nma/steps/avgshape/avgshape.py

```diff
@@ -84,24 +96,15 @@ def run(self, mesh_density=5, **kwargs):
             str(avg_data_dir / "avgshape.csv")
         )
 
-        remesh_dir = avg_data_dir / "remesh"
-        remesh_dir.mkdir(parents=True, exist_ok=True)
-        path_input_mesh = str(avg_data_dir / "avgshape.stl")
-        path_output = remesh_dir + "uniform_mesh"
-        uniform_trimesh(path_input_mesh, mesh_density, path_output)
-
         # Save path to avg shape in the manifest
         self.manifest = pd.DataFrame({
             "Label": "Average_nuclear_mesh",
             "AvgShapeFilePath": avg_data_dir / "avgshape.vtk",
-            "AvgShapeFilePathStl": avg_data_dir / "avgshape.stl",
-            "UniformMeshFilePathStl" : path_output + ".stl",
-            "UniformMeshFilePathBlend" : path_output + ".blend",
-            "UniformMeshVertices" : path_output + "_verts.npy",
-            "UniformMeshFaces" : path_output + "_faces.npy"
+            "AvgShapeFilePathStl": avg_data_dir / "avgshape.stl"
```

Comment:
Is STL format a Blender requirement? If not, can we stick with VTK?

Reply:
Great question - I know we have a lot of mesh file formats happening here which isn't desirable. But yes, stl is a Blender requirement for us to be able to shift between vtk and blender.

Reply:
I need to be able to import the mesh to blender and it looks like STL and PLY are the options (https://docs.blender.org/api/current/bpy.ops.import_mesh.html)

Reply:
That's fine. VTK is very flexible on that matter so I can save everything in STL from the beginning. I will add this as an issue to be addressed later.